packages/dcos-sdk/src/@types/react-router.d.ts

```diff
@@ -0,0 +1,15 @@
+declare module "react-router" {
```

Comment:
Can't we use the [official types](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/07352da21a909cb2786a40106aad1fea562d5973/types/react-router)

Reply:
They lack that specific type

Reply:
Thought about contributing this type back?

Reply:
Ah, understood :) Can we send an upstream PR? Got two in already in 1,5 weeks, they are pretty active. Just so that we can get rid of these types on the long run

Reply:
Sure, moreover contributing will be more like exporting something they have internally :)